examples/guided.py

```diff
@@ -308,31 +407,45 @@ def perform_installation(mountpoint):
 	Only requirement is that the block devices are
 	formatted and setup prior to entering this function.
 	"""
-	with archinstall.Installer(mountpoint, kernels=archinstall.arguments.get('kernels', 'linux')) as installation:
+	with archinstall.Installer(
+		mountpoint, kernels=archinstall.arguments.get('kernels', 'linux')
+	) as installation:
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.